[2025-03-23T17:58:35.947+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_data_hive manual__2025-03-23T17:58:08+00:00 [queued]>
[2025-03-23T17:58:35.964+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_data_hive manual__2025-03-23T17:58:08+00:00 [queued]>
[2025-03-23T17:58:35.966+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-03-23T17:58:35.986+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): load_data_hive> on 2025-03-23 17:58:08+00:00
[2025-03-23T17:58:35.995+0000] {standard_task_runner.py:57} INFO - Started process 14036 to run task
[2025-03-23T17:58:36.001+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_pipeline', 'load_data_hive', 'manual__2025-03-23T17:58:08+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmpgzsblagj']
[2025-03-23T17:58:36.005+0000] {standard_task_runner.py:85} INFO - Job 196: Subtask load_data_hive
[2025-03-23T17:58:36.071+0000] {task_command.py:415} INFO - Running <TaskInstance: fraud_detection_pipeline.load_data_hive manual__2025-03-23T17:58:08+00:00 [running]> on host 4ece88aec9a9
[2025-03-23T17:58:36.162+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='abelchaguermohamed@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='load_data_hive' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T17:58:08+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-23T17:58:08+00:00'
[2025-03-23T17:58:36.165+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-23T17:58:36.166+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec hive hive -f /home/hive_script.hql']
[2025-03-23T17:58:36.179+0000] {subprocess.py:86} INFO - Output:
[2025-03-23T17:58:38.280+0000] {subprocess.py:93} INFO - SLF4J: Class path contains multiple SLF4J bindings.
[2025-03-23T17:58:38.282+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hive/lib/log4j-slf4j-impl-2.6.2.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-23T17:58:38.283+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hadoop-2.7.4/share/hadoop/common/lib/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-23T17:58:38.284+0000] {subprocess.py:93} INFO - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[2025-03-23T17:58:38.284+0000] {subprocess.py:93} INFO - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
[2025-03-23T17:58:39.742+0000] {subprocess.py:93} INFO - 
[2025-03-23T17:58:39.743+0000] {subprocess.py:93} INFO - Logging initialized using configuration in file:/opt/hive/conf/hive-log4j2.properties Async: true
[2025-03-23T17:58:47.959+0000] {subprocess.py:93} INFO - OK
[2025-03-23T17:58:47.962+0000] {subprocess.py:93} INFO - Time taken: 7.86 seconds
[2025-03-23T17:58:48.229+0000] {subprocess.py:93} INFO - OK
[2025-03-23T17:58:48.229+0000] {subprocess.py:93} INFO - Time taken: 0.265 seconds
[2025-03-23T17:58:48.266+0000] {subprocess.py:93} INFO - OK
[2025-03-23T17:58:48.267+0000] {subprocess.py:93} INFO - Time taken: 0.035 seconds
[2025-03-23T17:58:48.296+0000] {subprocess.py:93} INFO - OK
[2025-03-23T17:58:48.297+0000] {subprocess.py:93} INFO - Time taken: 0.028 seconds
[2025-03-23T17:58:48.804+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-23T17:58:48.832+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=load_data_hive, execution_date=20250323T175808, start_date=20250323T175835, end_date=20250323T175848
[2025-03-23T17:58:48.849+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2025-03-23T17:58:48.850+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-03-23T17:58:49.396+0000] {email.py:282} INFO - Sent an alert email to ['abelchaguerm@gmail.com']
[2025-03-23T17:58:50.986+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-23T17:58:51.015+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
